src/Compiler/CodeGen/IlxGen.fs

```diff
@@ -5714,17 +5696,11 @@ and GenGenericParam cenv eenv (tp: Typar) =
             None
 
     let defaultConstructorConstraint =
-        tp.Constraints
-        |> List.exists (function
-            | TyparConstraint.RequiresDefaultConstructor _ -> true
-            | _ -> false)
+        tp |> HasConstraint _.IsRequiresDefaultConstructor
 
     let emitUnmanagedInIlOutput =
         cenv.g.langVersion.SupportsFeature(LanguageFeature.UnmanagedConstraintCsharpInterop)
-        && tp.Constraints
-           |> List.exists (function
-               | TyparConstraint.IsUnmanaged _ -> true
-               | _ -> false)
+        && tp |> HasConstraint _.IsIsUnmanaged
```

Comment:
`IsUnmanaged`?
```suggestion
        && tp |> HasConstraint _.IsUnmanaged
```

Reply:
This is autogenerated, I do not have control over it.
(the generated name is "Is" + name of the case)